assets/prototype/application/ui/components/display/entityListFilterBar/index.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External imports
+ */
+import * as React from 'react';
+import { withInstanceId } from '@wordpress/compose';
+import { IconButton } from '@wordpress/components';
+import { __ } from '@eventespresso/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Collapsible from './Collapsible';
+import './style.css';
+
+/**
+ * EntityListFilterBar
+ * a group of inputs for controlling how a list of entities is displayed
+ *
+ * @param {string} listId
+ * @param {string} instanceId
+ * @param {Object} entityFilters additional entity specific filters
+ * @param {string} searchText
+ * @param {number} perPage
+ * @param {string} view
+ * @return {Object} EntityListFilterBar
+ */
+const EntityListFilterBar = ({
```

Comment:
Had removed `index.js`

Reply:
I know, we had this conversation, and you said you wanted to have a clear separation in how folders are named vs how files are named; but we have this redundancy here:
![image](https://user-images.githubusercontent.com/1477453/70792234-74847780-1da1-11ea-932c-5eccabd09a90.png)

Reply:
If folder name would be `EntityListFilterBar` instead of `entityListFilterBar` and the `index.js` would be the current content of `EntityListFilterBar` we'd have a more cleaner path:
`import EntityListFilterBar from '../../../../../application/ui/components/display/EntityListFilterBar';`

Reply:
This organization is a bit related to collocation, in the sense that in `index.js` we might have the base/core/root component of a feature. The other subcomponents and hooks from the same folder will represent the hierarchical allegiance.